### DISCUSSION TITLE
Gleam support for JS DOM API

### DISCUSSION THREAD

**Discussion by yahia-berashish** [gleam-lang/gleam#3321] [Questions & support] [answered]
Hello 👋🏼 
I've been trying Gleam for some time now and I really like the language. 
I come from JS/TS web development and Gleam's ability to compile to type-safe JavaScript code was a huge factor for using the language. 
I recall seeing an example under the "Multilingual" section in [the Gleam website](https://gleam.run), this one specifically caught my attention: 
![image](https://github.com/gleam-lang/gleam/assets/74679313/84978e85-7429-4b63-98f3-76f86c690adf)
Is there an already existing Gleam library for interacting with the DOM API or is this example only for demonstration purposes? And if so, is a DOM API for Gleam relevant for a formal Gleam package, or is it an extension for the community to provide?

**Comment by lpil** [comment, marked as answer]
The Plinth package has bindings to these APIs

https://hexdocs.pm/plinth/plinth/browser/document.html

https://hexdocs.pm/plinth/plinth/browser/element.html
